db/go.mod

```diff
@@ -6,4 +6,7 @@ require (
 	github.com/dgraph-io/badger/v3 v3.2103.1
 	github.com/google/btree v1.0.0
 	github.com/stretchr/testify v1.7.0
+	github.com/tecbot/gorocksdb v0.0.0-20191217155057-f0fad39f321c
 )
+
+replace github.com/tecbot/gorocksdb => github.com/roysc/gorocksdb v0.0.0-20210804143633-c0bf0b3635e5 // FIXME
```

Comment:
What's the FIXME for? Are we pointing to  a specific commit for some reason?

Reply:
It's because it's pointing to my fork of the `tecbot` repo - they haven't responded to my [PR](https://github.com/tecbot/gorocksdb/pull/216) (no repo activity in 20 months actually). I can push a tag for the version though.